tool-box/schema-detection/runtime_linking.py

```diff
@@ -0,0 +1,91 @@
+def use_df():
+    import sys
+    import os
+    import digdag
+    sys.path.append('/home/td-user/.local/lib/python3.6/site-packages')
```

Comment:
This seems strange. It is for local, not server side one.

Reply:
No need to have this line for the latest image `digdag/digdag-python:3.7`